cfgov/jinja2/v1/_includes/macros/filters.html

```diff
@@ -178,11 +178,13 @@
    ========================================================================== #}
 {% macro _filters_form(filters, query, doc_type, options) %}
 
-<form class="{{ options.additional_classes }}"
-      method="{{ options.form_method }}"
-      {{ "action=" ~ options.action if options.action != '/' }}
+<form class="{{ options.additional_classes or '' }}"
+      method="{{ options.form_method or 'get' }}"
+      {{ "action=" ~ options.action if options.action != '/' else '' }}
       >
-    <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
+    {% if options.addition_classes and options.addition_classes == 'post' %}
```

Comment:
whoops i meant to write `{% if options.form_method and options.form_method == 'post' %}`. Sorry for the confusion